modules/deploying-models-using-multiple-gpu-nodes.adoc

```diff
@@ -0,0 +1,293 @@
++
+NOTE: If you have already configured a PVC, you can skip this step.
++
+[source]
+----
+kubectl apply -f - 
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: granite-8b-code-base-pvc
+spec:
+  accessModes:
+    - ReadWriteMany
+  volumeMode: Filesystem
+  resources:
+    requests:
+      storage: 50Gi
+  storageClassName: __<fileStorageClassName>__
+----
++
+
+. Download the model to the PVC. For example:
++
+[source]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: download-granite-8b-code
+  labels:
+    name: download-granite-8b-code
+spec:
+  volumes:
+    - name: model-volume
+      persistentVolumeClaim:
+        claimName: granite-8b-code-claim
+  restartPolicy: Never
+  initContainers:
+    - name: fix-volume-permissions
+      image: quay.io/quay/busybox@sha256:92f3298bf80a1ba949140d77987f5de081f010337880cd771f7e7fc928f8c74d 
+      command: ["sh"]
+      args: ["-c", "mkdir -p /mnt/models/granite-8b-code-base && chmod -R 777 /mnt/models"]
+      volumeMounts:
+        - mountPath: "/mnt/models/"
+          name: model-volume
+  containers:
+    - resources:
+        requests:
+          memory: 40Gi
+      name: download-model
+      imagePullPolicy: IfNotPresent
+      image: quay.io/modh/kserve-storage-initializer@sha256:83ab292e87f2d30c5db4f756dac7d99d0a0252a4563f784f30a2d75cbbd46330
+      args:
+        - 's3://$(BUCKET_NAME)/granite-8b-code-base/'
+        - /mnt/models/granite-8b-code-base
+      env:
+        - name: AWS_ACCESS_KEY_ID
+          value: <id>
+        - name: AWS_SECRET_ACCESS_KEY
+          value: <secret>
+        - name: BUCKET_NAME
+          value: <bucket>
+        - name: S3_USE_HTTPS
+          value: "1"
+        - name: AWS_ENDPOINT_URL
+          value: <AWS endpoint>
+        - name: awsAnonymousCredential
+          value: 'false'
+        - name: AWS_DEFAULT_REGION
+          value: <region>
+      volumeMounts:
+        - mountPath: "/mnt/models/"
+          name: model-volume
+----
++
+
+. Create the `vllm-multinode-runtime` ServingRuntime template as follows:
+
+.. Add the `vllm-multinode-runtime` ServingRuntime as a custom runtime:
++
+[source]
+----
+oc process vllm-multinode-runtime-template -n redhat-ods-applications|oc apply  -f -
+----
++
+
+.. From the namespace, define an InferenceService object in a YAML file called `vllm-multinode.yaml` with the following contents:
++
+[source]
+----
+... 
+ predictor:
+    model:
+      runtime: vllm-multinode-runtime
+      modelFormat:
+        name: vLLM
+      storageUri: pvc://granite-8b-code-base-pvc/hf/8b_instruction_tuned  
+    workerSpec: {} # Specifying workerSpec indicates that multi-node functionality will be used    
+----
++
+After you create the `vllm-multinode-runtime.yaml` file, the following configurations are added to the serving runtime:
```

Comment:
This part looks 2 things mingled. 
1. Creating `vllm-multinode-runtime.yaml` that is ServingRuntime
2. `oc process ..` . this is creating ServingRuntime with template.

Anyway, this line would explain that user can edit the following a new API for mult-node in the ServingRuntime as a  default value. But it can be overwritten by InferenceService.